src/services/google-analytics.ts

```diff
@@ -0,0 +1,75 @@
+import { VerificationFragment } from "@govtechsg/oa-verify";
+import { OpenAttestationDocument } from "@govtechsg/open-attestation";
+import { useEffect } from "react";
+import ReactGA from "react-ga4";
+
+const GTAG_ID = process.env.REACT_APP_GTAG_ID;
+
+export const useGoogleAnalytics = (): void => {
+  /* Initialise Google Analytics 4 if GTAG_ID is provided */
+  useEffect(() => {
+    try {
+      if (GTAG_ID?.startsWith("G-")) {
+        ReactGA.initialize(GTAG_ID);
+        ReactGA.send("pageview");
```

Comment:
need to provide the path ?

Reply:
Nope, won't be needed. As the GA script will automatically obtain the URL path for this `"pageview"` event.